test/known_issues/test-readline-interface-tty-readstream.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Done, I've left the comment with the description of what's trying to check the test (I think it doesn't hurts...).